schemas/session-replay/text-wireframe-schema.json

```
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "text-wireframe-schema.json",
+  "title": "TextWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a TextWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-schema.json"
+    },
+    {
+      "required": [
+        "text"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "text",
+          "readOnly": true
+        },
+        "text": {
+          "type": "string",
+          "description": "The text value of the wireframe.",
+          "readOnly": false
+        },
+        "textStyle": {
+          "type": "object",
+          "description": "The style of this text field.",
```

/nit - it's not only reserved for text fields. It can be used for "labels", "text areas", "segmented control titles" and many other platform-specific items - so better to stay platform agnostic.
```suggestion
          "description": "The style of the text.",
```